### ISSUE TITLE
Why do I have such high mAP% with YOLOv4-tiny?

### ISSUE THREAD

**Issue by stephanecharette** [AlexeyAB/darknet#6533]
I have a strange problem.  I'm using YOLOv4-tiny.  Project is very repetitive, assembly line images which look very similar to each other.  I have 3000 images with 50745 marked up objects, and I train with `max_batches` set to 20000.  But when I train, mAP% almost immediately goes to 100% and stays there for the duration of the training.  See this example (training not yet complete):

![image](https://user-images.githubusercontent.com/5061352/90973546-de7c2180-e4d7-11ea-82ba-a1869a29a8bd.png)

These are the lines I modify in the `[net]` section of the .cfg file:

```
flip=1
mosaic=1
hue=1.0
exposure=2.5
saturation=2.5
max_batches=20000
batch=64
subdivisions=8
width=832
height=352
```

Everything else in the .cfg file is left exactly as-is or is modified as described in readme.md, such as classes=1, the calculations for steps, etc.

Training is done with this command:

```
darknet detector -map -dont_show train /home/stephane/nn/Test/Test.data /home/stephane/nn/Test/Test.cfg
```

Is this normal for YOLOv4-Tiny?  Is there something I should be doing differently?

**Comment by stephanecharette** [comment]
> You should use different Training and Validation datasets, yes, divide 80% / 20%.

Yes, I'm using a random 80% of the images to train and the other 20% for validation.

**Comment by VisionEp1** [comment]
if you get 100% map, when the dataset is very easy, i dont see where to problem ist.
you can confirm by using detector test if you actually get 100% on your testset.
If so it's great isn't it?

**Comment by AlexeyAB** [comment]
@stephanecharette 
It can be if your dataset isn't hard enough or is small.

So you can:
* add more training/validation images
* use more/stronger data augmentation (increase parameters of data augmentation) - so your model will be more robust to different images and contexts in real-world
